Lib/asyncio/base_events.py

```diff
@@ -200,6 +200,28 @@ def _set_nodelay(sock):
         pass
 
 
+if hasattr(socket, 'TCP_QUICKACK'):
+    def _set_quickack(sock):
```

Comment:
You can merge the set/unset into a single set + 0/1 value. It's a private function anyway.